esp/platform/espcontext.cpp

```diff
@@ -503,37 +510,90 @@ class CEspContext : public CInterface, implements IEspContext
         return m_txSummary.get();
     }
 
-    virtual void addTraceSummaryValue(LogLevel logLevel, const char *name, const char *value)
+    virtual void addTraceSummaryValue(LogLevel logLevel, const char *name, const char *value, const unsigned int group = TXSUMMARY_GRP_OPS1)
```

Comment:
I was told in a recent review that default parameters are no longer acceptable. Check on that before adding this and others.